charts/psmdb-db/README.md

```diff
@@ -243,6 +243,16 @@ The chart can be customized using the following configurable parameters:
 | `users.roles.role.name`        | Name of the MongoDB role assigned to the user. As [built-in roles](https://www.mongodb.com/docs/manual/reference/built-in-roles/#built-in-roles), so [custom roles](https://github.com/mongodb/mongodb-kubernetes-operator/blob/master/docs/deploy-configure.md#define-a-custom-database-role) are supported | `""`                     |
 | `users.roles.role.db`          | Database that the MongoDB role applies to                                                                                                                                                                                                                                                                    | `""`                     |
 | |
+| `roles.role`                      | Name of the custom role. | `""`                           |
+| `roles.db`                      | Database in which you want to store the user-defined role. | `"admin"`                           |
+| `roles.authenticationRestrictions.clientSource` | Array of IP addresses or CIDR blocks from which users assigned this role can connect.MongoDB servers reject connection requests from users with this role if the requests come from a client that is not present in this array. | `""`                           |
+| `roles.authenticationRestrictions.serverAddress` | Array of IP addresses or CIDR blocks to which users assigned this role can connect.MongoDB servers reject connection requests from users with this role if the client requests to connect to a server that is not present in this array. | `""`                           |
+| `roles.privileges.actions` |  Name of the role. Valid values are built-in roles. | `[]`                           |
+| `spec.roles.privileges.resource.db` |  Database for which the privilege `spec.security.roles.privileges.actions` apply. An empty string ("") indicates that the privilege actions apply to all databases. | `""`                           |
```

Comment:
Please remove the "spec." prefix in the parameters and descriptions.

Also, just a tip to format the table, try turning word wrap off in your code editor(if you are not doing it yet). The lines are huge, but I hope this will make the table more readable to edit.

Reply:
[c050068](https://github.com/percona/percona-helm-charts/pull/376/commits/c05006860ffec5f65f28945e17a8b60e0b4f866d)